rustic-doc.el

```diff
@@ -311,9 +331,11 @@ NO-DL is primarily used for development of the filters."
                              (lambda (_p)
                                (message "Finished converting docs for std"))
                              "std")
+  (sleep-for 3)
```

Comment:
Maybe wait for the process to finish instead ? Or what's reason that we have to wait here ?

Reply:
This was to prevent minibuffer messages to instantly be overwritten by other minibuffer messages. I didn't see a way to avoid it at the time (even if I agreed that adding sleeps here and there is not great), but it's changed now.

Reply:
Sometimes sleeping is necessary, unfortunately.